resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -126,6 +126,8 @@ package object config extends Logging {
       .stringConf
       .createOptional
 
+  private[spark] val KUBERNETES_DRIVER_ENV_KEY = "spark.driverEnv."
```

Comment:
Ah never mind - this is confusing because the driver env requires a new Kubernetes-specific configuration but the executors will use an existing configuration from spark-core.